docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,83 @@
+
+#################################
+Access your Apps via SSH
+#################################
+
+**Audience:** Users who want to debug their Flow and Work in a the runtime environment.
+
+----
+
+**********************************************************
+Generating a new SSH key and configure ssh-agent
```

Comment:
Titles should be verbs, see style guide: https://www.notion.so/lightningai/Check-page-structure-4c4e660c70b949a7aa75763e6d0955e9